src/ethereum/scripts/gen_genesis_config.go

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
Would you mind putting this and [`b2t.sh`](https://github.com/ethereum/eth1.0-specs/blob/master/b2t.sh) into `<repository-root>/scripts/...`?

I'd like to keep `src` as pure python, if possible :P